core/new-gui/src/app/common/service/workflow-persist/workflow-persist.service.ts

```diff
@@ -103,7 +103,7 @@ export class WorkflowPersistService {
    * @param workflow
    * @private
    */
-  private static parseWorkflowInfo(workflow: Workflow): Workflow {
+  public static parseWorkflowInfo(workflow: Workflow): Workflow {
```

Comment:
since multiple files are using this util function now, I suggest we move it to a common/util location.